utils/getSessionUDT.js

```diff
@@ -0,0 +1,7 @@
+const getSessionUDT = (sample) => {
+  const content = JSON.parse(sample.content);
+  const annotation = JSON.parse(sample.annotation);
+  return { ...content, annotation };
+};
```

Comment:
function name isn't good, it's not returning a "UDT JSON Object" (AKA "Dataset Object"), it's return a "Sample Object". Rename to `getSampleObject`.

Reply:
done